src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] colors = Color.values();
+
+    public static String getRandomColor() {
```

Comment:
The method 'getRandomColor' should not be static according to the checklist. Static methods are generally considered bad practice in this context. You should create an instance of the class to call this method.